File: common/src/main/java/com/ssblur/scriptor/block/ScriptorBlockTags.java
package com.ssblur.scriptor.block;

import com.ssblur.scriptor.ScriptorMod;
import net.minecraft.core.registries.Registries;
import net.minecraft.tags.TagKey;
import net.minecraft.world.level.block.Block;

public class ScriptorBlockTags {
  public static final TagKey<Block> DO_NOT_PHASE = TagKey.create(Registries.BLOCK, ScriptorMod.location("do_not_phase"));
}

File: common/src/main/kotlin/com/ssblur/scriptor/block/ScriptorBlockTags.kt
package com.ssblur.scriptor.block

import com.ssblur.scriptor.ScriptorMod
import net.minecraft.core.registries.Registries
import net.minecraft.tags.TagKey
import net.minecraft.world.level.block.Block

object ScriptorBlockTags {
    @JvmField
    val DO_NOT_PHASE: TagKey<Block> = TagKey.create(Registries.BLOCK, ScriptorMod.location("do_not_phase"))
}
